### PR TITLE
Fixed VEBA UI image + Documentation Updates

### LINKED ISSUE #337: Annotate Knative Service/Triggers with app:veba-ui
**Describe the bug**
With the introduction of the Sockeye service which includes a Knative Trigger which does not contain a filter, this unintentionally causes the VEBA UI to miss-behaves as it expects trigger to always contain filter. Simple fix is to apply app:veba-ui label just like we do for secrets, to only display things created from within the VEBA UI and/or manually label as such

### PR DESCRIPTION
**Pull request by lamw**
Signed-off-by: William Lam <wlam@vmware.com>

## Summary

This change ensures that that the VEBA UI only displays Knative Service/Triggers which have been annotated with `app-veba-ui" and ensures non-customer facing Knative Service/Triggers are not accidentally displayed or negatively impact the UI service. This change also includes documentation updates to reflect the required changes for users who wish to have their Knative Service/Trigger show up in VEBA UI

## Pull Request Checklist
🚨 Please review the [guidelines for contributing](https://vmweventbroker.io/community) to this repository.

- [x] Please ensure that you are making a pull request against the **Development** branch
- [ ] Please use the `WIP` keyword in the title of your PR if you are not ready for review
- [x] Please ensure that you have opened a Github Issue if you are resolving/fixing a problem
- [x] Please ensure that you have [signed](https://help.github.com/en/github/authenticating-to-github/signing-commits) all commits and that you have [squashed](https://medium.com/@slamflipstrom/a-beginners-guide-to-squashing-commits-with-git-rebase-8185cf6e62ec) all relevant commmits related to your change
- [ ] Please make sure that you have tested your change locally by successfully [building and deploying the VMware Event Broker Appliance](https://vmweventbroker.io/kb/contribute-appliance) and/or [building and deploying VMware Event Router](https://vmweventbroker.io/kb/contribute-eventrouter)
- [ ] Please include any relevant screenshots and/or output as part of your testing
- [x] Please include any documentation updates that is applicable for your changes

## Change Type

What types of changes does your code introduce to the VMware Event Broker Appliance?

_Put an `x` in all boxes that apply_

Please check the type of change your PR introduces:
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] Feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Documentation changes
- [ ] Other (please describe):

## Resolved Issues

Closes #337

## Testing Verification

* Verified VEBA UI functions correctly w/the new Sockeye service

## Additional Information

N/A

### CODE REVIEW

File: docs/kb/intro-functions.md [hunk 1]
```diff
@@ -62,6 +64,8 @@ The value of this field:
 - should not contain special characters, e.g. "$" or "/"
 - should represent the intent of the function, e.g. "tag" or "tagging"
 
+`veba-ui`: The Kubernetes label that is required for the VMware Event Broker Appliance UI to display manually deployed Knative Service
```
Comment (embano1) [inline docs/kb/intro-functions.md:67]:
```suggestion
`app: veba-ui`: The Kubernetes label that is required for the VMware Event Broker Appliance UI to display a manually deployed Knative Service
```

File: docs/kb/intro-functions.md [hunk 2]
```diff
@@ -105,6 +111,8 @@ The value of this field:
 - should not contain special characters, e.g. "$" or "/"
 - should represent the intent of the trigger, e.g. "tag" or "tagging"
 
+`veba-ui`: The Kubernetes label that is required for the VMware Event Broker Appliance UI to display manually deployed Knative Trigger
```
Comment (embano1) [inline docs/kb/intro-functions.md:114]:
```suggestion
`app: veba-ui`: The Kubernetes label that is required for the VMware Event Broker Appliance UI to display a manually deployed Knative Trigger
```

File: docs/kb/intro-functions.md [hunk 3]
```diff
@@ -138,6 +148,8 @@ The value of this field:
...
```
Comment (embano1) [inline docs/kb/intro-functions.md:151]: See comments/suggest change above

File: docs/kb/intro-functions.md [hunk 4]
```diff
@@ -117,6 +125,8 @@ kind: Trigger
 metadata:
   name: veba-ps-echo-trigger
   namespace: vmware-functions
```
Comment (embano1) [inline docs/kb/intro-functions.md:119]: Wanna keep namespace?

File: examples/knative/powershell/kn-ps-echo/README.md [hunk 5]
```diff
@@ -75,4 +75,12 @@ For testing purposes, the `function.yaml` contains the following annotations, wh
 annotations:
   autoscaling.knative.dev/maxScale: "1"
   autoscaling.knative.dev/minScale: "1"
+```
+
+# Step 4 - Undeploy
+
+```bash
+# Deploy function
```
Comment (embano1) [inline examples/knative/powershell/kn-ps-echo/README.md:83]:
```suggestion
# Undeploy function
```

File: examples/knative/powershell/kn-ps-slack/README.md [hunk 6]
```diff
@@ -78,4 +78,12 @@ For testing purposes, the `function.yaml` contains the following annotations, wh
 annotations:
   autoscaling.knative.dev/maxScale: "1"
   autoscaling.knative.dev/minScale: "1"
+```
+
+# Step 4 - Undeploy
+
+```bash
+# Deploy function
```
Comment (embano1) [inline examples/knative/powershell/kn-ps-slack/README.md:86]:
```suggestion
# Undeploy function
```

**Review by embano1 — CHANGES_REQUESTED** [review]

**Comment by lamw** [comment]
Thanks for the quick review @embano1, all feedback addressed!

**Review by embano1 — APPROVED** [review]
LGTM

**Pull request merged**